### PR TITLE
Add mapping page for Cloud Foundry provider

### PR DESCRIPTION
**Pull request by fifthposition**
This adds a mapping page (resources and operations) for the Cloud Foundry provider.

### CODE REVIEW

File: reference/providers/cf.md
Comment (dorbin) [inline reference/providers/cf.md]: @fifthposition 
I assume by "app" you're not referring to a Spinnaker application, right?

**Review by dorbin — COMMENTED** [review]
Thanks for the new doc, @fifthposition !

In general, can you link terminology (like route. app, foundation, space) to relevant docs on cloudfoundry.org or elsewhere?

**Comment by fifthposition** [comment]
Thanks @dorbin! Updated based on your feedback.

**Review by dorbin — COMMENTED** [review]
Thanks, @fifthposition! LGTM, pending review by @lwander .

**Comment by fifthposition** [comment]
Hi @dorbin @lwander! Any update here? :-)

File: reference/providers/cf.md
Comment (lwander) [inline reference/providers/cf.md]: I would reorder these, rollback starts the prior server group before stopping the current one.

File: reference/providers/cf.md
Comment (lwander) [inline reference/providers/cf.md:40]: Please insert an extra newline after the headers for consistency with other pages. Also some markdown rendering fails without the extra line.

File: reference/providers/cf.md
Comment (fifthposition) [reply reference/providers/cf.md]: Fixed. Good catch. Thanks!

File: reference/providers/cf.md
Comment (fifthposition) [reply reference/providers/cf.md]: Done. Thank you!

**Comment by fifthposition** [comment]
@lwander @dorbin Updated (thanks!). OK to go now? :)

**Review by lwander — APPROVED** [review]

**Pull request merged**
